ivy/functional/frontends/tensorflow/math.py

```diff
@@ -325,6 +325,15 @@ def squared_difference(x, y, name=None):
     return ivy.square(ivy.subtract(x, y))
 
 
+@with_supported_dtypes(
+    {"2.9.0 and below": ("float16", "float32", "float64", "complex64", "complex128")},
+    "tensorflow",
+)
```

Comment:
`tensorflow` actually does support `bfloat16` in inputs. You can verify that [here](https://www.tensorflow.org/api_docs/python/tf/math/sin). Maybe you can add that here 🙂

Reply:
Thank you sir. I have made the desired changes.